src/arithmetic_impls.rs

```diff
@@ -0,0 +1,294 @@
+use crate::{decimal::CalculationResult, ops, Decimal};
```

Comment:
`src/decimals.rs` has more than two thousand line so in order to keep my mental health, I moved all arithmetic implementations to this module.